projects/deno.land/package.yml

```diff
@@ -19,17 +19,41 @@ runtime:
     DENORT_BIN: '{{prefix}}/bin/denort'
 
 dependencies:
-  x86-64:
+  darwin/x86-64:
+    # FIXME try removing this after new builds are available
     llvm.org: 17 # libunwind
 
 build:
   script:
+    # deno does not need llvm to build on darwin+x86-64, and if it is present,
+    # deno will be linked to its libunwind, which then causes deno to need llvm
+    # in runtime.
+    - |
+      if test "{{hw.platform}}+{{ hw.arch }}" != "darwin+x86-64"; then
+        set -o allexport
+        source <(pkgx +llvm.org@17)
+        set +o allexport
+      fi
```

Comment:
ok, but this makes llvm17 a build dep for everything but darwin+x86-64, and it's already brought in to darwin+x86-64 by the run-time requirement...

Reply:
I didn't realize runtime requirements were added to build environment implicitly.

Anyway, if I remove the runtime dependency, the build won't even start on darwin+x86-64.

Reply:
Let's see if tests passes now on darwin+x86-64 with the libunwind check.

Reply:
Test is passing: https://github.com/pkgxdev/pantry/actions/runs/8502469906/job/23286685906?pr=5751#step:9:133

Reply:
Ok, @jhheider would you like me to do anything else? Worst case it's better to have it only as a runtime dependency on darwin then on all platforms.